### PR TITLE
Add support for release notes

### PR DESCRIPTION
**Pull request by nhumrich**

### CODE REVIEW

**Review by TheMcMurder — APPROVED** [review]
Lgtm

File: tests/api/test_api.py [hunk 1]
```diff
@@ -165,6 +165,34 @@ def test_get_audit_logs(client):
     assert response.status_code == 200
 
 
+def test_create_release_note(client):
+    response = client.post('/api/release_notes',
+                           json={'title': 'obi-wan',
+                                 'body': 'has the high ground',
+                                 'feature': 'bobbytables'})
```
Comment (TheMcMurder) [inline tests/api/test_api.py:172]: :heart_eyes:

File: tests/api/test_api.py [hunk 2]
```diff
@@ -165,6 +165,34 @@ def test_get_audit_logs(client):
...
+    results = client.get('/api/envs/production/release_notes')
+    id_ = 0
+    for rn in results.json()['release_notes']:
+        if rn['title'] == 'obi-wan':
+            id_ = rn['id']
+
+    response = client.delete(f'/api/release_notes/{id_}')
+    assert response.status_code == 204
+
+
```
Comment (TheMcMurder) [inline tests/api/test_api.py]: It might be a good idea to test that removing a toggle also removes the associated toggle from a release.

Comment (nhumrich) [reply tests/api/test_api.py]: good call

Comment (nhumrich) [reply tests/api/test_api.py]: updated

**Review by TheMcMurder — APPROVED** [review]
:octopus:

**Pull request merged**